base_export_manager/views/ir_exports.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Due to the renaming and the reindentation, this appears as a whole new file. It's better to leave as is (although the indentation is not correct), but we should improve the diff size.